hack/charts/cluster-api-operator/values.yaml

```diff
@@ -1,12 +1,30 @@
 ---
 # ---
 # Cluster API provider options
-core: ""
-bootstrap: ""
-controlPlane: ""
-infrastructure: ""
-ipam: ""
-addon: ""
+core: {}
+  # name: ""
+  # namespace: ""  # Optional
+  # version: ""  # Optional
```

Comment:
This is currently not a list because:
> [Note: Only one CoreProvider can be installed at the same time on a single cluster.](https://cluster-api-operator.sigs.k8s.io/03_topics/03_basic-cluster-api-provider-installation/01_installing-core-provider)

But maybe it should be a singleton list just to align it with the others?

Reply:
I think it can stay as is, can't think of the point it being a list tbh